pkg/app/src/components/OrganisationCard/modules/tileCard.tsx

```diff
@@ -0,0 +1,76 @@
+import { useExtensionContext } from 'src/provider/extension/modules/context'
+import { NavLink } from 'src/components/NavLink/navLink'
+import { useTheme } from '@mui/material/styles'
+import { Typography, Card, Box, CardHeader, CardContent, Avatar } from '@mui/material'
+import { Person, Key, KeyOff, Check } from '@mui/icons-material'
+
+const gateway = 'https://ipfs.gamedao.co/gateway/'
+const toLink = '/app/organisations/'
+
+export const TileCard = (props) => {
+	const { selectedAccount } = useExtensionContext()
+	const theme = useTheme()
+	const bgPlain = { backgroundColor: theme.palette.grey[500_16] }
+	const title = props.data.name.length > 17 ? `${props.data.name.slice(0, 17)} ...` : props.data.name
+	const address = selectedAccount?.account.address
+
+	const SubHeader = () => {
```

Comment:
Should be its own module.
Is inefficent, with each rerender the subhader is redefined.